src/handlers/expiry/providers/DateProvider.ts

```diff
@@ -100,8 +100,8 @@ export class DateProvider {
   public static isBetweenDates(
     fromDate: string | number | Date,
     toDate: string | number | Date,
-    compareFromDate: Date,
-    compareToDate: Date
+    compareFromDate: string | number | Date,
+    compareToDate: string | number | Date
```

Comment:
Why did you need to allow storing dates as numbers here? I would avoid it

Reply:
All numbers have been removed from DateProvider in 18783.